rock_changed.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
All other helper scripts are stored in `tools` directory. Do you think this one can be moved there too?